UC-Berkeley-Extension/global
# Microsite test

## Asana Ticket(s)
- See ticket [Determine Sub-theme render options](https://app.asana.com/0/1174970560648905/1194057934900278/f).

## Description

- Sets up a taxonomy for use as the sub-theme indicator.


## Testing instructions

- Run `composer install` from the project root.
- Run `../scripts/refresh.sh` or `../scripts/setup.sh` depending on whether you want a new build or not.
- You should have:

  - 2 images (in /web/sites/default/files/2020-10/)
  - 2  media items that contain those images
  - 2 Taxonomy terms in the 'school' vocabulary
  - 1 Article node assigned to the school vocabulary.
  - 1 Views page node assigned to the school vocabulary.

## Remaining Tasks

- [x] Figure out how we import default taxonomies here.
- [x] Figure out how to set the taxonomy context for pages that are not node -- e.g. Views. It may be that we *always* embed Views within node pages to get around this issue.
